backend/tests/Tests/Execution.Tests.fs

```diff
@@ -589,6 +596,15 @@ let testMatchPreview : Test =
           (okVarRhsVarId, "rhs", er (DString "y"))
           (okVarRhsStrId, "str", er (DString "ok: ")) ]
 
+      t
+        "tuple match"
+        (eTuple (eInt 2) (eStr "sample") [])
```

Comment:
@StachuDotNet it was a little hard to fully understand what's going on in this test, but I think this should do the trick